src/cookbook/design/themes.md

```diff
@@ -9,42 +9,75 @@ js:
 
 <?code-excerpt path-base="cookbook/design/themes"?>
 
+{{site.alert.note}}
+  This recipe uses Flutter's support for [Material 3][] and
+  the [google_fonts][] package.
+{{site.alert.end}}
+
+[Material 3]: /ui/design/material
+[google_fonts]: {{site.pub-pkg}}/google_fonts
+
 To share colors and font styles throughout an app, use themes.
-You can either define app-wide themes, or use `Theme` widgets
-that define the colors and font styles for a particular part
-of the application. In fact,
-app-wide themes are just `Theme` widgets created at
-the root of an app by the `MaterialApp`.
 
-After defining a Theme, use it within your own widgets. Flutter's
-Material widgets also use your Theme to set the background
-colors and font styles for AppBars, Buttons, Checkboxes, and more.
+You can define app-wide themes.
+You can extend a theme to change a theme style for one component.
+Each theme defines the colors, type style, and other parameters
+applicable for the type of Material component.
+
+If a widget doesn't use a specific theme,
+the visual properties fall back to the main theme.
```

Comment:
I like that you added this discussion earlier.

But the following list contradicts the statement that "the visual properties fall back to the main theme". Could we just drop this sentence and have just the "Flutter applies styling in the following order:"?